plugin/trino-mongodb/src/test/java/io/trino/plugin/mongodb/MongoServer.java

```diff
@@ -29,10 +31,16 @@ public MongoServer()
     }
 
     public MongoServer(String mongoVersion)
```

Comment:
The usage of this constructor is only 2 places. I would change this constructor instead of adding a new constructor.